src/app/profiles/export-dialog/export-dialog.component.ts

```diff
@@ -34,14 +28,14 @@ import { Router, RouterModule } from '@angular/router'
   styleUrl: './export-dialog.component.scss'
 })
 export class ExportDialogComponent implements OnInit {
-  data = inject(MAT_DIALOG_DATA)
```

Comment:
I think we still need this right?

Reply:
I dont see it used, do you?

Reply:
Because the dialog uses it, I thought we might need it when it's removed. It's not being used anywhere, so not required.